src/backend/app/Console/Kernel.php

```diff
@@ -59,9 +59,9 @@ class Kernel extends ConsoleKernel {
      * @return void
      */
     protected function schedule(Schedule $schedule) {
-        $schedule->command('reports:city-revenue weekly')->weeklyOn(1, '3:00');
-        $schedule->command('reports:city-revenue monthly')->monthlyOn(1, '3:00');
-        $schedule->command('reports:outsource')->monthlyOn(1, '3:30');
+        $schedule->command('reports:city-revenue weekly')->weeklyOn(1, '3:00')->appendOutputTo(storage_path('logs/cron.log'));
```

Comment:
I'm curious. Is this for debugging, or is it really needed here?

Reply:
For debugging, I couldn't find any logs of it while investigating because by default Laravel sends the output to `/dev/null`

Reply:
Alright, understood. Let's remove these debug statements for now tho before merge this one into `main`.

I think it should be a separate task to make the logs appear on STDOUT/STDERR.

Reply:
Updated